bash_completionsV2.go

```diff
@@ -115,6 +116,14 @@ __%[1]s_process_completion_results() {
                 __%[1]s_debug "No space directive not supported in this version of bash"
             fi
         fi
+        if (((directive & shellCompDirectiveKeepOrder) != 0)); then
+            if [[ $(type -t compopt) == builtin ]]; then
+                __%[1]s_debug "Activating keep order"
+                compopt -o nosort
```

Comment:
the `nosort` was added with bash 4.4 according to the answer to https://unix.stackexchange.com/questions/215937/disable-sorting-of-compreply-in-bash-complete-function

When I use bash 4.2 the script throws an error:
```
bash4-4.2$ ./prog orderfromshell [tab][tab]
bash4-4.2$ ./prog orderfromprogram bash4: compopt: nosort: invalid option name
```
Can we protect against that?